heat/core/dndarray.py

```diff
@@ -2299,6 +2299,101 @@ def __or__(self, other):
         """
         return arithmetics.bitwise_or(self, other)
 
+    def pad(self, pad, mode="constant", value=0):
```

Comment:
I don't think we need to provide a method  self.pad(), I would take it out and just leave the function.